Makefile

```diff
@@ -105,3 +105,13 @@ fmt:
 	find . '(' -name '*.h' -o -name '*.c' ! -path './vendor/*' ')'  -exec clang-format -i {} \+
 	find . -name '*.go' ! -path './vendor/*' -exec gofmt -s -w {} \+
 	git diff --exit-code
+
+
+.PHONY: dbuild
+dbuild:
```

Comment:
is dbuild short for docker build? We may want a different name here,. unless this is standardized across projects

Reply:
Well, it's mostly a [copy&paste from your partner repo](https://github.com/containers/podman/blob/master/Makefile#L268-L278) and I would like to keep the stuff in sync as I mostly build it together.